### DISCUSSION TITLE
Extended Site Settings for Category and Commerce Product

### DISCUSSION THREAD

**Discussion by vandres** [craftcms/cms#11729] [Q&A] [answered]
In Entries, it is possible to set the availability and default visibility for sites. 

![image](https://user-images.githubusercontent.com/1436383/182384617-0aff2038-08d1-455b-bc14-6f321d06965e.png)

I would wish, to have this feature also for element type Category and Craft Commerce Product.

Alternatively, I s there an easy way to implement that feature by myself. I could easily have some configuration in the Globals. I just don't see a way, to overwrite the default `Category::getSupportedSites()`

**Comment by brandonkelly** [comment, marked as answer]
See #8478 for an existing discussion about that feature for Categories.
